index.html

```diff
@@ -10,7 +10,7 @@
   <!-- load the testing dependencies -->
   <script src='./dependencies/chai.js'></script>
   <script src='./dependencies/describe-it.js'></script>
-
+  <link rel="stylesheet" href="style.css">
```

Comment:
there is a link on style.css , see the line 8

Reply:
yes thank you ! i will delete.